content/en/standards/reproducibility/_index.md

```diff
@@ -12,4 +12,68 @@ weight: 2
 These standards promote the reuse of model code and reproducibility of model results.
 {{% /standards-preamble %}}
 
-Reproduciblity is a cornerstone of scientific research, especially for transparent evaluation of research claims. Equally important is the potential for scientific procedures to be reused and modified in order to carry out new research. While clear documentation is essential to reusability, new technologies also offer the potential to increase the reproducibility and reusability of model code. A critical metric of reproducibility is whether a model can consistently reproduce the published results that claim to have been generated by the model. Reproducibility and reusability can also be enhanced if model code is accompanied by explicit workflows, or workflow scripts.
+In this document we are adopting the [National Academies of Science, Engineering and Medicine 2019 report on Reproducibility and Replicability in Science](https://doi.org/10.17226/25303) definitions of reproducibility and replicability.
+
+> Reproducibility means obtaining consistent computational results using the same input data, computational steps, methods, code, and conditions of analysis.  Replicability means obtaining consistent results across studies aimed at answering the same scientific question, each of which has obtained its own data.  
+
+Reproducibility is a cornerstone of scientific research, especially for transparent evaluation of research claims. Equally important is the potential for scientific procedures to be reused and modified in order to carry out new research. While clear documentation is essential to reusability, new technologies also offer the potential to increase the reproducibility and reusability of model code. A critical metric of reproducibility is whether a model can consistently reproduce the published results that claim to have been generated by the model. Reproducibility and reusability can also be enhanced if model code is accompanied by explicit workflows that demonstrate how the model generates and transforms its output data into its published findings.
+
+
+### Minimal Standards 
+
+This set of minimal standards can be easily adopted by journals to ensure that submitted publications meet baseline reproducibility and reusability requirements. The high level goal is for an independent reviewer to be able to easily download, build, and execute a computational model and verify that it meets the reproducibility claims stated by the model author(s).
```

Comment:
> easily adopted by journals to ensure that submitted publications meet baseline reproducibility and reusability requirements

Perhaps we should bring this up to the JOSS editor on the call to see how much trouble it would be to get this process in place once vetted.